packages/x-charts/src/ScatterChart/ScatterChart.test.tsx

```diff
@@ -101,19 +105,27 @@ describe('<ScatterChart />', () => {
         <ScatterChart {...config} disableVoronoi series={[{ id: 's1', data: config.dataset }]} />
       </div>,
     );
-    const svg = document.querySelector<HTMLElement>('svg')!;
     const marks = document.querySelectorAll<HTMLElement>('circle');
 
-    fireEvent.pointerEnter(svg); // Trigger the tooltip
-    fireEvent.pointerMove(marks[0]); // Only to set the tooltip position
-
-    fireEvent.pointerEnter(marks[0]);
-    let cells = document.querySelectorAll<HTMLElement>('.MuiChartsTooltip-root td');
-    expect([...cells].map((cell) => cell.textContent)).to.deep.equal(['', '', '(0, 10)']);
-
-    fireEvent.pointerEnter(marks[4]);
-    cells = document.querySelectorAll<HTMLElement>('.MuiChartsTooltip-root td');
-    expect([...cells].map((cell) => cell.textContent)).to.deep.equal(['', '', '(5, 5)']);
+    await user.pointer([
+      // Only to set the tooltip position
+      { target: marks[0] },
+    ]);
+
+    let cells: NodeListOf<HTMLElement> = [] as any;
+    await waitFor(() => {
+      cells = document.querySelectorAll<HTMLElement>('.MuiChartsTooltip-root td');
+      expect([...cells].map((cell) => cell.textContent)).to.deep.equal(['', '', '(0, 10)']);
+    });
```

Comment:
Why do we need to `waitFor`? Is it because of the animation? 

Would it make sense to wait for the tooltip to show instead and then assert synchronously?

Reply:
True, I changed to waitFor because the fully sync was flaky. Your suggestion should probably work.